lms/tasks/email_digests.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+from typing import List
+
+from lms.services import DigestService, MailchimpService
+from lms.tasks.celery import app
+
+
+@app.task
+def send_instructor_email_digests(h_userids: List[str], since: str, until: str) -> None:
+    """
+    Generate and send instructor email digests to the given users.
+
+    The email digests will cover activity that occurred in the time period
+    described by the `since` and `until` arguments.
+
+    :param h_userids: the h_userid's of the instructors to email
+    :param since: the beginning of the time period as an ISO 8601 format string
+    :param until: the end of the time period as an ISO 8601 format string
+    """
+    # Caution: datetime.fromisoformat() doesn't support all ISO 8601 strings!
+    # This only works for the subset of ISO 8601 produced by datetime.isoformat().
+    since = datetime.fromisoformat(since)
+    until = datetime.fromisoformat(until)
```

Comment:
...and these are strings because I don't think you can actually pass `datetime`'s to a Celery task directly.